Tasks/AzureFunctionOnKubernetesV1/task.json

```
@@ -0,0 +1,192 @@
+        {
+            "name": "appName",
+            "type": "string",
+            "required": true,
+            "label": "Application Name",
+            "groupName": "commands",
+            "helpMarkDown": "Application Name. The Kubernetes objects created use this name. This should follow Kubernetes naming conventions for resource names."
+        },
+        {
+            "name": "functionRootDirectory",
+            "type": "filePath",
+            "label": "Function root directory",
+            "required": false,
+            "groupName": "commands",
+            "helpMarkDown": "Function root directory. Should contain host.json. Docker build and push is performed from this directory."
+        },
+        {
+            "name": "waitForStability",
+            "type": "boolean",
+            "label": "Wait for stability",
+            "groupName": "commands",
+            "defaultValue": "true",
+            "helpMarkDown": "Wait for the Kubernetes objects to reach the desired state."
+        },
+        {
+            "name": "arguments",
+            "type": "string",
+            "label": "Arguments",
+            "groupName": "commands",
+            "helpMarkDown": "Pass arguments to command. Ex:<br>--no-docker --service-type NodePort"
+        }
+    ],
+    "dataSourceBindings": [
+        {
+            "target": "kubernetesCluster",
+            "endpointId": "$(azureSubscriptionEndpoint)",
+            "endpointUrl": "{{{endpoint.url}}}/subscriptions/{{{endpoint.subscriptionId}}}/resourceGroups/$(azureResourceGroup)/providers/Microsoft.ContainerService/managedClusters?api-version=2017-08-31",
+            "resultSelector": "jsonpath:$.value[*]",
+            "resultTemplate": "{{{name}}}"
+        },
+        {
+            "target": "azureResourceGroup",
+            "endpointId": "$(azureSubscriptionEndpoint)",
+            "endpointUrl": "{{{endpoint.url}}}/subscriptions/{{{endpoint.subscriptionId}}}/providers/Microsoft.ContainerService/managedClusters?api-version=2017-08-31",
+            "resultSelector": "jsonpath:$.value[*]",
+            "resultTemplate": "{{{ #extractResource id resourcegroups}}}"
+        }
+    ],
+    "instanceNameFormat": "Func on K8s - deploy",
+    "execution": {
+        "Node10": {
+            "target": "src//run.js"
+        }
+    },
+    "messages": {
+        "AllContainersNotInReadyState": "All the containers are not in a ready state.",
+        "CouldNotDeterminePodStatus": "Could not determine the pod's status due to the error: %s",
+        "CouldNotDetermineServiceStatus": "Could not determine the service %s status due to the error: %s",
+        "FileContentSynced": "Synced the file content to the disk. The content is %s.",
+        "KubectlClientVersion": "Kubectl Client Version",
+        "KubectlServerVersion": "Kubectl Server Version",
+        "NoDataWrittenOnFile": "No data was written into the file %s",
+        "ServiceExternalIP": "service %s external IP is %s",
```

Should this message starts from uppercase?

```suggestion
        "ServiceExternalIP": "Service %s external IP is %s",
```